iothub/device/src/DeviceAuthenticationWithTokenRefresh.cs

```diff
@@ -11,32 +11,26 @@ namespace Microsoft.Azure.Devices.Client
     /// </summary>
     public abstract class DeviceAuthenticationWithTokenRefresh : AuthenticationWithTokenRefresh
     {
-        private const int DefaultTimeToLiveSeconds = 1 * 60 * 60;
-        private const int DefaultBufferPercentage = 15;
-
-        /// <summary>
-        /// Initializes a new instance of the <see cref="DeviceAuthenticationWithTokenRefresh"/> class using default
-        /// TTL and TTL buffer time settings.
-        /// </summary>
-        /// <param name="deviceId">Device Identifier.</param>
-        public DeviceAuthenticationWithTokenRefresh(string deviceId)
-            : this(deviceId, DefaultTimeToLiveSeconds, DefaultBufferPercentage)
-        {
-        }
+        internal const int DefaultTimeToLiveSeconds = 1 * 60 * 60;
+        internal const int DefaultBufferPercentage = 15;
```

Comment:
I updated these variables to be internal since I need them for a unit test and I did not want to copy over the values manually.